CEXT-4373: Update the oope shipping methods documentation (#330)
AdobeDocs/commerce-extensibility: src/pages/starter-kit/checkout/use-cases.md

You can add shipping methods to the checkout process by using [webhooks](../../webhooks/index.md).

> To be able to add shipping methods, you must first [run a script to automatically create shipping carriers](./configure.md#create-shipping-carriers) or [to create shipping carriers manually](./shipping-reference.md#shipping-api-reference) using REST Api. Only shipping methods with the registered carriers for them will be available in the checkout process.

Reviewer: ```suggestion
To add shipping methods, you must [run a script to automatically create shipping carriers](./configure.md#create-shipping-carriers) or [create shipping carriers manually](./shipping-reference.md#shipping-api-reference) using the REST API. Only shipping methods with registered carriers are available in the checkout process.
```
